tests/test_coverity.py

```diff
@@ -13,9 +14,16 @@
 
 class TestCoverityWarnings(TestCase):
     def setUp(self):
+        os.environ['MIN_COV_WARNINGS'] = '0'
+        os.environ['MAX_COV_WARNINGS'] = '0'
         self.warnings = WarningsPlugin(verbose=True)
         self.warnings.activate_checker_name('coverity')
 
+    def tearDown(self):
+        for var in ('MIN_POLY_WARNINGS', 'MAX_POLY_WARNINGS'):
+            if var in os.environ:
+                del os.environ[var]

```

Comment:
Lets start using `unittest.mock.patch.dict` to set envvar on a per test case or test suite basis. https://adamj.eu/tech/2020/10/13/how-to-mock-environment-variables-with-pythons-unittest/

Reply:
Resolved in 70ab119de8cb158dfc0f2db9fc0ed60f4fbb1de9